README.md

```diff
@@ -97,6 +97,27 @@ bulkinvalidaddresses
 Rows which are successfully processed will be added to `PROCESSED_invalid_addresses_*.csv` and errored rows be appended to `ERROR_invalid_addresses_*.csv` with the corresponding error details written to `ERROR_DETAIL_invalid_addresses_*.csv`.
 
 
+### Find Invalid Address Case IDs from UPRN File
+Run Book - https://collaborate2.ons.gov.uk/confluence/display/SDC/Find+Invalid+Address+Case+ID%27s+by+UPRN+-+ADDRESS_DELTA
+
+When we receive a file of UPRNs for cases that have been identified as invalid addresses, this feature will call the Case API against the UPRNs provided and generate a new file in the bulk invalid address bucket with the name `invalid_addresses_invalid_uprn.csv`.
```

Comment:
README here claims the command just leaves the file in the bucket, it is actually running the bulk processor on it too.